pkg/reconciler/pipelinerun/resources/pipelinerunstate.go

```diff
@@ -324,6 +334,32 @@ func (facts *PipelineRunFacts) GetSkippedTasks() []v1beta1.SkippedTask {
 	return skipped
 }
 
+// GetPipelineTaskStatus returns the status of a PipelineTask depending on its taskRun
+func (facts *PipelineRunFacts) GetPipelineTaskStatus(ctx context.Context) map[string]string {
+	// construct a map of context.pipelineRun.tasks.<pipelineTask>.status and its state
```

Comment:
please update comment to use current syntax

Reply:
Fixed 🙏